app/lib/submission_builder/ty2022/states/az/documents/state1099_g.rb

```diff
@@ -30,7 +30,7 @@ def document
                 xml.RecipientSSN recipient.ssn
                 xml.RecipientName recipient.full_name
                 xml.RecipientUSAddress do
-                  xml.AddressLine1Txt form1099g.recipient_street_address
+                  xml.AddressLine1Txt form1099g.recipient_address_line1
```

Comment:
@tofarr Should the apartment go in a `AddressLine2Txt` element instead of being concatenated into line 1?